### DISCUSSION TITLE
How to compile from source?

### DISCUSSION THREAD

**Discussion by macvenez** [markqvist/Reticulum#538] [Help & Questions] [answered]
Hi, I'm trying to compile from source the code for Heltec V3.
I couldn't find any information to build from source so I inspired from the Makefile doing:
`make prep-esp32` and 
`make firmware-heltec32_v3`
first command goes fine and installs everything with no problems, the second one could't finish correctly because of some errors. The files mentioned in the error log are effectively not there, there are only the .cpp and not the .cpp.o.
Has anyone else experienced this? Am I missing something?
I am running Arch Linux 6.10.7
[compile.log](https://github.com/user-attachments/files/16829256/compile.log)

**Comment by jacobeva** [comment, marked as answer]
Perhaps try compiling my newer community edition instead? https://github.com/liberatedsystems/RNode_Firmware_CE
If you still have issues, please contact me here.

**Reply by macvenez** [reply]
Thanks! It's working!

**Comment by jacobeva** [comment]
Great!
